### ISSUE TITLE
Godot 3 support with import plugin

### ISSUE THREAD

**Issue by PiOverFour** [ndee85/coa_tools#108]
Hi @ndee85, I know that you stated in #84 you had no interest in continuing development on the Godot importer/exporter, but for what it's worth I converted your importer to Godot 3, in much the same way as @muart did also in #84, only using an [import plugin](http://docs.godotengine.org/en/latest/tutorials/plugins/editor/import_plugins.html).

I can make a PR if you are interested, meanwhile the code is in [this branch](https://github.com/PiOverFour/coa_tools/tree/master).

You mentioned that you would look at Godot 3.1's features for mesh animation and maybe write a tscn exporter in Blender; do you have any update in that regard?

**Comment by rektuiemforameme** [comment]
Do the importer and exporter support mesh deformation? I exported from coatools and imported using your modified importer, but deformations don't seem to work.

**Comment by PiOverFour** [comment]
No, when I updated the importer, 2D meshes with [internal vertices](http://docs.godotengine.org/en/latest/tutorials/animation/2d_skeletons.html#internal-vertices) weren't yet supported, so I didn't implement deformation.

**Comment by rektuiemforameme** [comment]
Bummer. My project is pretty much totally based on mesh deformation and I don't want to use the dragonbones export. Thanks anyway

**Comment by ndee85** [comment]
The old godot exporter only supports normal sprite export. No mesh deformation. Right now the only way is to go with the dragonbones export and use this module in godot.
https://github.com/sanja-sa/gddragonbones
